controller/createJobController.ts

```diff
@@ -0,0 +1,29 @@
+import { createIncrementalCompilerHost } from 'typescript';
+import { createJob } from '../model/createJob';
+import { Application, Request, Response } from "express";
+const createJobService = require('../service/createJobService')
+
+module.exports = function(app: Application){
+
+    app.get('/create-job-roles', async (req: Request, res: Response) =>{
+        res.render('create-job-roles')
```

Comment:
you're not getting the bands and capabilities from the API to populate the drop-down lists

Reply:
> you're not getting the bands and capabilities from the API to populate the drop-down lists

The form in views  that encapsulates those lists makes a post request to the database so they should be connected to the database